assets/js/analyzer.js

```diff
@@ -1,5 +1,6 @@
 var analyzers = {}, analyzerData = {};
 var currentAnalyzerRequest;
+var THRESHOLD_REQUEST_LENGTH_FOR_ANALYZE = 2020;
```

Comment:
@sushain97 , Please have a look! I have modified it to a generalised function. I have added the functions to be used across scripts in the `/* global */` comment at the top. However, I believe I have not done that correctly yet!